src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -63,9 +63,14 @@ final case class SuccessfulExecutionHandle(outputs: CallOutputs, events: Seq[Exe
   override val result = SuccessfulBackendCallExecution(outputs, events, returnCode, hash, resultsClonedFrom)
 }
 
-final case class FailedExecutionHandle(throwable: Throwable, returnCode: Option[Int] = None) extends ExecutionHandle {
+final case class FailedExecutionHandle(throwable: Throwable, returnCode: Option[Int] = None, events: Seq[ExecutionEventEntry] = Seq.empty) extends ExecutionHandle {
```

Comment:
Will do. I noticed that events are not written in the database for a Call that fails the workflow though (NonRetryable failure)